src/Chainweb/CutDB.hs

```diff
@@ -416,10 +417,7 @@ startCutDb config logfun headerStore payloadStore cutHashesStore = mask_ $ do
                 Right hm -> unsafeMkCut v
                     <$> case _cutDbParamsInitialHeightLimit config of
                         Nothing -> return hm
-                        Just h -> limitCutHeaders wbhdb h hm
-                            -- FIXME: this requires that the pact db is deleted because it
-                            -- interfers with rewind limit. As a fix temporarily disable
-                            -- rewind limit during initialization.
+                        Just h -> limitCutHeaders wbhdb (avgCutHeightAt v h) hm
```

Comment:
Maybe we should change the argument of `limitCutHeaders` from cut height to block height?